authgear/authgear-server
# SAML: Redirect user to Auth UI

ref DEV-1726

How to test:
Use this site: https://sptest.iamshowcase.com/authnrequest

Note that the response part is not done yet so you cannot complete the flow.

Also addressed comments in https://github.com/authgear/authgear-server/pull/4591

Note that I've made some adjustments to the errors.
